src/main/java/edu/gatech/cs2340/trydent/TrydentException.java

```diff
@@ -12,5 +12,8 @@
     public TrydentException(String message) {
         super(message);
     }
+    public TrydentException(String message, Exception nestedException) {
```

Comment:
I don't like to have line breaks between related methods, but that's not a strong opinion. I'll take a poll at our meeting today.